public/getting_started/installation_guide_no_vm.html

```diff
@@ -165,7 +165,7 @@ <h1>Installation Guide - No VM</h1>
 
 </p>
                     <ul>
-                        <li class="bullet_list" value="1">PHP v7.1.x<br />            apt-get install php7.0-curl php7.0-gd php7.0-gmp php7.0-intl php7.0-mcrypt php7.0-pgsql php-redis php7.0-xml php7.0-mbstring php7.0-bz2</li>
+                        <li class="bullet_list" value="1">PHP v7.2.x<br />            apt-get install php7.0-curl php7.2-curl php7.2-gd php7.2-gmp php7.2-intl php7.2-pgsql php7.2-redis php7.2-xml php7.2-mbstring php7.2-bz2 php7.2-json php7.2-bcmath php7.2-zip php7.2-cli</li>

```

Comment:
This is NOT true. "Minimum requirements" are and will be 7.1.
This can be "recommended requirements" etc.

Please separate those two. We must not be lying here.

Reply:
@dereuromark 
Hm, then I got this wrong. What I understood was "following needs to be installed for the software to run". Not "following versions are the minimum ones supported". 
I think in that case it would be nice to clarify that in the docs with a new section?

So I think it's clear the CODE requires php 7.1 or more, but I still think documentation should still suggest installing latest supported version. 
I agree on adding new section called "Recommended"  so it's separated from "Minimal", though.

Composer package already states that.
```
"php": ">=7.1",
```
So just clarifying the docs with new section is fine, I guess.

Reply:
@dereuromark added the new section, feel free to review (also specified jenkins in recommended, because that's the only one that worked. will be later fixed I guess)